lld/test/ELF/incompatible.s

```diff
@@ -1,3 +1,16 @@
+# REQUIRES: aarch64
+# RUN: rm -rf %t
+# RUN: touch %t.call_graph 
+# RUN: touch %t.profdata 
```

Comment:
Minor nit, but I realized that you can use `/dev/null` in place of these files and remove the `touch` commands.